File: MT-Api/src/main/java/br/com/mucatour/model/travel/entity/Payment.java
package br.com.mucatour.model.travel.entity;

import javax.persistence.Column;
import javax.persistence.FetchType;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;

import lombok.Builder;
import lombok.Data;

@Builder
@Data
public class Payment {
    
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column(name = "user_id", nullable = false)
    private Long userId;

    @ManyToOne(fetch = FetchType.EAGER, optional = false)
    @JoinColumn(name = "travel_id", referencedColumnName = "id")
    private Travel travel;
}

File: MT-Web/src/models/payment.js
export default class Payment {

    constructor(
        title,
        dueDate,
        createdAt,
        userId,
        travelId,
        list
        

    ) {
        this.title = title;
        this.dueDate = dueDate;
        this.createdAt = createdAt;
        this.userId = userId;
        this.travelId = travelId;        
        this.list = list;
    }
    
}